src/Orleans.Core.Abstractions/CodeGeneration/InvokeMethodRequest.cs

```diff
@@ -68,10 +68,25 @@ public enum InvokeMethodOptions
         // Transactional method options. 
         // NOTE: keep in sync with TransactionOption enum.
         // Suppress is default, so no need for flag for it
-        TransactionCreateOrJoin = 0x200,
-        TransactionCreate = 0x400,
-        TransactionMandatory = 0x800,
-        TransactionSupported = 0x1000,
-        TransactionNever = 0x2000,
+        TransactionFlags = 0xE00,
```

Comment:
`TransactionOptionsMask` (since this isn't a valid value and this is a bit mask and they're no longer flags)